src/Aether/Devices/Drivers/Sgp4x.cs

```diff
@@ -0,0 +1,174 @@
+            // 0x4B = One or more test failed
+            return testResultWithCRC[0] == 0xD4;
+        }
+
+        /// <summary>
+        /// Gets the raw VOC measurements from the device.
+        /// </summary>
+        /// <param name="relativeHumidityValue">The relative humidity value expressed as a percentage.</param>
+        /// <param name="temperatureValue">The temperature value expressed in degrees celsius.</param>
+        /// <returns>The raw VOC measurement value from the sensor. If an error occurred, it will be <see langword="null"/>.</returns>
+        /// <remarks>If default relative humidity and temperature values are supplied, humidity compensation will be disabled.</remarks>
+        public VolumeConcentration? GetVOCRawMeasure(ushort relativeHumidityValue = 50, short temperatureValue = 25)
+        {
+            if (relativeHumidityValue > 100)
+            {
+                throw new ArgumentOutOfRangeException(nameof(relativeHumidityValue), relativeHumidityValue, "A relative humidity percentage value must be between 0 and 100");
+            }
+
+            if (temperatureValue < -45 || temperatureValue > 130)
+            {
+                throw new ArgumentOutOfRangeException(nameof(temperatureValue), temperatureValue, "A temperature value must be between -45 and 130");
+            }
+
+            Span<byte> writeBuffer = stackalloc byte[8];
+            Span<byte> readBuffer = stackalloc byte[3];
+
+            // Write start measure VOC raw command
+            writeBuffer[0] = 0x26;
+            writeBuffer[1] = 0x0F;
+
+            // Write default humidity value + CRC (0x80, 0x00, [CRC], 0xA2)
+            Sensirion.WriteUInt16BigEndianAndCRC8(writeBuffer.Slice(2, 3), (ushort)Math.Round((double)(relativeHumidityValue * 65535.00 / 100.00), MidpointRounding.AwayFromZero));
+            
+            // Write default temperature value + CRC (0x66, 0x66, [CRC], 0x93)
+            Sensirion.WriteUInt16BigEndianAndCRC8(writeBuffer.Slice(5, 3), (ushort)((temperatureValue + 45) * 65535.00 / 175.00));
+            
+            // Transmit command
+            _device.Write(writeBuffer);
+
+            Thread.Sleep(30);
+
+            _device.Read(readBuffer);
+
+            // Read the results and validate CRC
+            ushort? readValue = Sensirion.ReadUInt16BigEndianAndCRC8(readBuffer);
+
+            if(readValue is null)
+            {
+                return null;
+            }
+
+            Console.WriteLine("VOC RAW: {0}", readValue.Value);
+
+            Sgp4xAlgorithm.VocAlgorithmParams algoParams = new Sgp4xAlgorithm.VocAlgorithmParams();
+
+            Sgp4xAlgorithm.VocAlgorithm_init(algoParams);
+
+            int vocValue = -1;
+            Sgp4xAlgorithm.VocAlgorithm_process(algoParams, readValue.Value, out vocValue);
+
+            Console.WriteLine("VOC INDEX: {0}", vocValue);
+
+            return new VolumeConcentration(vocValue, UnitsNet.Units.VolumeConcentrationUnit.PartPerBillion);
```

Comment:
the VOC index is not a traditional measurement but some proprietary thing. Probably just return `int?`.

Reply:
Wrote custom unit for this. I can return int? if it is determined to be better.